### PR TITLE
add compatibility to unsafe_read/write additions to base julia in v0.5

### PR DESCRIPTION
**Pull request by vtjnash**

### CODE REVIEW

**Comment by malmaud** [comment]
Thanks!

File: src/ssl.jl [hunk 1]
```diff
@@ -123,55 +124,72 @@ end
...
         (Ptr{Void},), ctx.data)
     ctx.isopen = true
+    nothing
 end
 
-function Base.write(ctx::SSLContext, msg::Vector{UInt8})
-    n = 0
-    N = length(msg)
-    while n < N
+if Base.VERSION < v"0.5-"
```
Comment (tkelman) [inline src/ssl.jl]: this needs to be more specific about the version number

**Comment by malmaud** [comment]
@vtjnash care to step in here?

**Comment by tkelman** [comment]
Jameson, please respond. Which base PR did this API correspond to?

**Comment by vtjnash** [comment]
https://github.com/JuliaLang/julia/pull/14766

**Pull request merged**